Content.Client/UserInterface/ParticleAccelerator/ParticleAcceleratorControlMenu.cs

```diff
@@ -108,15 +108,15 @@ public ParticleAcceleratorControlMenu(ParticleAcceleratorBoundUserInterface owne
             _offButton = new Button
             {
                 ToggleMode = false,
-                Text = "Off",
+                Text = Loc.GetString("generic-off"),
```

Comment:
While `On` and `Off` look like states you can be in, as opposed to verbs... some languages have stative-verbs!
I'd err on the side of making these specific and not generic.